core/linker/default.h

```diff
@@ -83,6 +83,11 @@ SECTIONS
         *(.ramfunc.*)
         *(.data)
         *(.data.*)
+
+        /* The __uvisor_export_table must be placed at the end of flash to work
+         * correctly. */
+        KEEP(*(.uvisor_export_table))
```

Comment:
`uvisor_export_table` is data. It's a table of exports from uVisor. So, in that sense it makes sense to put the table into the data section.

Additionally, the `uvisor_export_table` must be located at the end of flash. Placing it into the end of the data section places it at the end of flash.